src/powershell/install.sh

```diff
@@ -242,14 +242,22 @@ if [ "${use_github}" = "true" ]; then
     install_using_github
 fi
 
-# If PowerShell modules are requested, loop through and install 
+# If PowerShell modules are requested, loop through and install
 if [ ${#POWERSHELL_MODULES[@]} -gt 0 ]; then
     echo "Installing PowerShell Modules: ${POWERSHELL_MODULES}"
     modules=(`echo ${POWERSHELL_MODULES} | tr ',' ' '`)
     for i in "${modules[@]}"
     do
-        echo "Installing ${i}"
-        pwsh -Command "Install-Module -Name ${i} -AllowClobber -Force -Scope AllUsers" || continue
+        IFS=':' read -ra module_parts <<< "$i"
```

Comment:
```suggestion
        IFS='==' read -ra module_parts <<< "$i"
```

Rethinking about this, let's use `==` as the delimiter. We can sync with Python's way of pinning to a version.

Reply:
Done